tests/phpunit/tests/query/wpLoop.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * Tests to make sure looping through posts using the wp_loop() function works properly.
+ *
+ * @ticket 48193
+ *
+ * @group  query
+ */
+class Tests_WP_Loop extends WP_UnitTestCase {
+
+	/**
+	 * Create test posts in database.
+	 *
+	 * @param WP_UnitTest_Factory $factory
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		$factory->post->create( [ 'post_title' => 'Post 1' ] );
+		$factory->post->create( [ 'post_title' => 'Post 2' ] );
+		$factory->post->create( [ 'post_title' => 'Post 3' ] );
```

Comment:
Double check that the cleanup of these posts happens automatically as part of the WP test framework.

Reply:
Yes, it does.